ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/machine/MachineServiceClientImpl.java

```diff
@@ -53,24 +52,6 @@ protected MachineServiceClientImpl(@RestContext String restContext,
     }
 
     @Override
-    public Promise<MachineDto> getMachine(@NotNull final String workspaceId,
-                                          @NotNull final String machineId) {
-        return asyncRequestFactory.createGetRequest(baseHttpUrl + workspaceId +
-                                                    "/machine/" + machineId)
-                                  .header(ACCEPT, APPLICATION_JSON)
-                                  .loader(loaderFactory.newLoader("Getting info about machine..."))
-                                  .send(dtoUnmarshallerFactory.newUnmarshaller(MachineDto.class));
-    }
-
-    @Override
-    public Promise<List<MachineDto>> getMachines(@NotNull String workspaceId) {
-        return asyncRequestFactory.createGetRequest(baseHttpUrl + workspaceId + "/machine")
-                                  .header(ACCEPT, APPLICATION_JSON)
-                                  .loader(loaderFactory.newLoader("Getting info about bound machines..."))
-                                  .send(dtoUnmarshallerFactory.newListUnmarshaller(MachineDto.class));
-    }
-
-    @Override
     public Promise<Void> destroyMachine(@NotNull final String workspaceId,
```

Comment:
@vparfonov We still have this action https://github.com/eclipse/che/blob/master/plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/actions/DestroyMachineAction.java#L32
So I can not to remove this method